author.hbs

```diff
@@ -113,8 +120,30 @@
                     </div>
                 </div>
 
+                {{#has slug="adam-kudrna"}}
+                    <a
+                        href="https://www.adamkudrna.cz/konzultace"
```

Comment:
Tak a FG už dělá oficiálně i promo. 🙈 To tam budou moct mít i další autoři? (Neptám se kvůli sobě, to opravdu nehrozí. 🙂 )

Reply:
V tuhle chvíli to míním sobecky pro sebe, ale jestli bude poptávka, asi proč ne.